src/forwarder.ts

```diff
@@ -189,6 +192,15 @@ function shouldSubscribe(resource: any, forwarderConfigs: ForwarderConfigs) {
   ) {
     return false;
   }
+
+  // If the log group does not belong to our list of handlers, we don't want to subscribe to it
+  if (
+    resource.Properties.LogGroupName.startsWith("/aws/lambda/") &&
+    !handlers.some(({ name }) => resource.Properties.LogGroupName === `/aws/lambda/${name}`)
```

Comment:
Hmmm, this might not work because people can override the log group name. I might switch it back to comparing the Logical ID of the group